tests/ZfcRbacTest/Factory/AuthorizationServiceDelegatorTest.php

```diff
@@ -95,6 +100,36 @@ public function testAuthorizationServiceIsInjectedWithDelegator()
         $this->assertEquals($authorizationService, $decoratedInstance->getAuthorizationService());
     }
 
+    public function testAuthorizationServiceIsInjectedWithDelegatorV3()
```

Comment:
maybe I don't see but where is the difference with `testAuthorizationServiceIsInjectedWithDelegator()` test?